app.js

```diff
@@ -24,38 +24,49 @@ let rocket = {
   velocity: 0,
   element: document.querySelector("#rocket"),
 }
-  let lvlComplete = {
-    element: document.querySelector('#levelComplete')
-  }
 
 function startGame() {
   initKeybordMovement();
-  enemy1.element.addEventListener('click', fireGatlingEnemyOne);
-  enemy2.element.addEventListener('click', fireGatlingEnemyTwo);
-  enemy3.element.addEventListener('click', fireGatlingEnemyThree);
+  enemies.forEach(addGatling);
   gameFps = setInterval(Step, 1000 / fps);
 }
 
+function addGatling (enemy) {
+  function handleEnemyClick (event) {
+    fireGatlingEnemy(event, enemy);
+  }    
+  enemy.element.addEventListener('click', handleEnemyClick);
+}
+
 function Step() {
-  renderEnemy(enemyHealth50text, healthBar50text, enemyHealth50, healthBar50, enemy1);
-  renderEnemy(enemyHealth100text, healthBar100text, enemyHealth100, enemyHealth100.element, enemy2);
-  renderEnemy(enemyHealth200text, healthBar200text, enemyHealth200, healthBar200, enemy3);
-  enemy1.x = moveEnemy(enemy1.x, enemy1.velocity);
-  enemy2.x = moveEnemy(enemy2.x, enemy2.velocity);
-  enemy3.x = moveEnemy(enemy3.x, enemy3.velocity);
-  enemyHealth50.x = moveEnemy(enemyHealth50.x, enemyHealth50.velocity);
-  enemyHealth100.x = moveEnemy(enemyHealth100.x, enemyHealth100.velocity);
-  enemyHealth200.x = moveEnemy(enemyHealth200.x, enemyHealth200.velocity);
-  enemyHealth50text.x = moveEnemy(enemyHealth50text.x, enemyHealth50text.velocity);
-  enemyHealth100text.x = moveEnemy(enemyHealth100text.x, enemyHealth100text.velocity);
-  enemyHealth200text.x = moveEnemy(enemyHealth200text.x, enemyHealth200text.velocity);
+  //renderEnemy(enemyHealth50text, healthBar50text, enemyHealth50, healthBar50, enemy1);
+  //renderEnemy(enemyHealth100text, healthBar100text, enemyHealth100, enemyHealth100.element, enemy2);
+  //renderEnemy(enemyHealth200text, healthBar200text, enemyHealth200, healthBar200, enemy3);
+  // changed to
+  enemies.forEach(el => {renderEnemy(el)})
+  //enemy1.x = moveEnemy(enemy1.x, enemy1.velocity);
+  //enemy2.x = moveEnemy(enemy2.x, enemy2.velocity);
+  //enemy3.x = moveEnemy(enemy3.x, enemy3.velocity);
+  //changed to
+  enemies.forEach(el => {moveEnemy({el})})
```

Comment:
You can simply write `enemines.forEach(moveEnemy);`

Reply:
Secondly, you are passing `el` inside and object: `{el}`.

And you are receiving it as `{el}` object in the `moveEnemy` function as well.

Change `function moveEnemy({el}) {` to `function moveEnemy(el) {`.
And use ` enemines.forEach(moveEnemy);`.

This should have the same effect as your code, but with less symbols :)